conf/lookup/BCD-Port.csv

```diff
@@ -11452,7 +11451,7 @@ PortId,CountryCode,PortType,PortCode,KeyWords,Name,Name-cy,Priority,Region,Locat
 31870,UK,2,FIE,,Fair Isle Airport - FIE,Fair Isle Airport - FIE,,1,NULL,1,
 31879,UK,2,FAB,,Farnborough Airport - FAB,Farnborough Airport - FAB,,1,NULL,1,
 31961,UK,2,BHD,,George Best Belfast City Airport - BHD,George Best Belfast City Airport - BHD,5,2,NULL,1,
-31969,UK,2,GLW,,Glasgow International Airport - GLW,Glasgow International Airport - GLW,5,1,NULL,1,
+31969,UK,2,GLW,Bowling,Glasgow International Airport - GLW,Glasgow International Airport - GLW,5,1,NULL,1,

```

Comment:
Duplicates 1203,UK,2,GLA,,Glasgow Airport - GLA,Glasgow Airport - GLA,,1,NULL,0,

and has the wrong Code, delete this line